T7_1/Infrastructure/LibraryLoader.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Homework_7_1.DAL;
+using Homework_7_1.Entities;
+
+namespace Homework_7_1.Infrastructure
+{
```

Comment:
Please implement AbstractFactory to create Library for eBooks and PaperBooks separately, not in one method. This method is not effective because maybe you will need to create , for example , eBooks catalog only